sway-lib-std/src/vec.sw

```diff
@@ -142,3 +145,84 @@ impl<T> Vec<T> {
         self.len == 0
     }
 }
+
+/// TODO: Add proper errors
+
+pub struct StorageVec<V> {}
+
+impl<V> StorageVec<V> {
```

Comment:
It would be very useful to have a `.contains()` method

Reply:
Similar to with `vec`, `contains()` isn't strictly required for vector functionality, so can be left to a future PR. @SwayStar123 can you file an issue for additional methods to implement for storage vec?